packages/clay-link/src/ClayLink.js

```diff
@@ -26,6 +26,15 @@ ClayLink.STATE = {
 	 */
 	ariaLabel: Config.string(),
 
+	/**
+	 * Aria Expanded attribute for the anchor element.
+	 * @instance
+	 * @memberof ClayLink
+	 * @type {?bool|undefined}
+	 * @default undefined
+	 */
+	areaExpanded: Config.bool(),
```

Comment:
Is `areaExpanded` a typo error? Should it be `ariaExpanded`?

Reply:
Yeah, went unnoticed 😅. Thanks!!